site/guide/4-requests.md

```diff
@@ -678,7 +678,12 @@ struct Task<'r> {
 fn new(task: Json<Task<'_>>) { /* .. */ }
 ```
 
-See the [JSON example](@example/serialization/src/json.rs) on GitHub for a complete example.
+! NOTE: JSON support requires the `json` feature flag to be enabled in Rocket
+
+  Rocket has intentionally chosen to place JSON support, as well as a few other formats
+  behind feature flags.
```

Comment:
Can we add a link to <https://api.rocket.rs/master/rocket/#features> somewhere in this sentence, which explains the available features and shows how to enable them?

Reply:
Yes, absolutely. I added a sentence about the other features, and linked to the api reference. I also added a small code example to demonstrate how to enable the feature in Cargo.toml.